noxfile.py

```diff
@@ -103,3 +104,11 @@ def docs(session):
     if os.path.exists("_build"):
         shutil.rmtree("_build")
     session.run("sphinx-build", "-W", ".", "_build/html")
+
+
+@nox.session(python=["3.8"])
+def spelling(session):
+    subprocess.call(["sudo", "apt-get", "update"])
```

Comment:
If someone tries to run this nox session on a non Debian Derived Linux or macOS or Windows, this will not work. Is there another way to get this functionality?

Reply:
Thanks for pointing this out @sigmavirus24 - I’ll do some research and see what options
there are.